content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,124 @@
+
+
+## Set the starting level
+
+Before you create a project game release layout, specify the starting level that the Game Launcher will load first.
+
+1. Open the `autoexec.cfg` file in your project's directory and add the following line. This sets the starting level by using `Loadlevel` and providing a path to the level. The path must be relative to the `Cache/pc` directory. In this example, the level asset is `mainmenu.spawnable`.
+
+    ```
+    Loadlevel levels/mainmenu/mainmenu.spawnable
+    ```
+
+## Process assets
+
+The Asset Processor or Asset Processor Batch are responsible for processing source assets into product assets, which the Game Launcher consumes at runtime. 
+
+1. Process your project's assets by following either one of the following instructions.
+    
+    - Use **CMake** to build the Asset Processor Batch and its dependencies by specifying the `MyProject.Assets` target. After building, this command runs Asset Processor Batch and processes the assets.
+
+        ```cmd
+        cmake --build build/windows_vs2019 --target MyProject.Assets --config profile -- /m
+        ```
+
+    - Run your project's Asset Processor to begin processing the assets.
+
+<!-- [Question] Is it accurate to say that both methods can complete this task? Is one method recommended? If so, why? -->
+
+{{< caution >}} Ensure that the Asset Processor or Asset Processor Batch processes all of your assets without failure. {{< /caution >}}
+
+
+## Create a project game release layout
+
+You can create a project game release layout by using a non-monolithic or monolithic build. A monolithic release layout only contains files that're necessary to run the project, such as the Game Launcher, packaged assets, and dependent `.dll` files. A non-monolithic release layout contains the same files needed to run the project, as well as the objects that you use for development such as tools, shader compilers, and all `.dll` files.
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory that will contain your release layout.
+
+- Packages an `engine.pak` file out of the product assets in your `Cache/pc` directory. 
+
+- Builds the Game Launcher and other supporting files.
+
+
+### Monolithic project
+
+1. Set up a new CMake build directory, `windows_mono`, for your monolithic build in your project directory. We recommend keeping this build directory separate from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
+
+    ```cmd
+    cmake -B build/windows_mono -S. -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_MONOLITHIC_GAME=1
+    ```
+
+2. Run the CMake build configuration in your monolithic build directory. This generates a Game Launcher and packages your project's assets in the `Cache` directory into `engine.pak`.
```

Comment:
Do we need to re-iterate here that if that any .pak file over 4GiB likely will fail to load?  I'm not sure if we have a workaround for this yet, but the real way to do this would be to use the asset bundler to generate multiple .pak files and load those.